### DISCUSSION TITLE
Nuxt2 Security Only Maintenance in 2024

### DISCUSSION THREAD

**Discussion by Austio** [nuxt/nuxt#23342] [Ideas]
Hey Nuxt Team, opening a discussion for changing the roadmap status of Nuxt2 from End of Life in December to Security Maintenance Mode only.  Functionally that would mean that instead of closing down new relelases, Nuxt2 would still accept PRs from the community and maintainers for Security Patches only.

### Why Do We Need More Time

Migrating to Nuxt 3 is a quite large task for large projects due to the number of core breaking changes with libraries and apis.  Large production apps focused on delivering features generally can't just rewrite everything, so instead we need an iterative path where we can opt in to features gradually to reduce the chances of breaking things for users of our site.

Our team along with many others have been waiting for some of the items outline in the [Migration Guide](https://github.com/nuxt/nuxt/discussions/19895) to be complete so that we had an easier migration path to Nuxt 3.  We tried nuxt bridge 3 times over the last year and each time it was not ready yet to help on large migrations.  Daniel Roe seems to have it close in the latest [release candidate branch](https://github.com/nuxt/bridge/tree/v3.0.0-rc.1) and it was one of the 4 major tasks noted by atinux in [this Nuxt Bridge issue](https://github.com/nuxt/bridge/issues/891).

### Why Does Accepting Security Patches Matter

Putting Nuxt 2 in End of Life will expose many real world apps at a higher level of risk to being compromised.  Further, companies who require insurance will be out of compliance due to running end of life software in production.

Simply deciding on the posture of "Security Maintenance Mode" only until Nuxt Bridge is ready and teams have had a chance to use it will mitigate the risk here.

### What is a Security Patch

Security Patches are those related to the Nuxt Framework.  We cannot control the Vue ecosystem, but we can respond to and fix Security Vulnerabilities that are reported privately or publicly on the Nuxt repo.  This is a net positive for the community and something that I personally will help with during the interrim.

### How Long

I propose that we keep this repo in Security Maintenance Mode Only for 4 months after the release candidate from [Daniel Roe is released](https://github.com/nuxt/bridge/tree/v3.0.0-rc.1).

**Comment by danielroe** [comment]
Thank you for this proposal.

I've now announced that the new EOL for Nuxt 2 will be June 30, 2024. This is documented on the [Nuxt 2 website](https://v2.nuxt.com/lts/) and you can see it also in the [release notes for Bridge RC1](https://github.com/nuxt/bridge/releases/tag/v3.0.0-rc.1).

I very much appreciate your forbearance with the delay in releasing Nuxt Bridge, and hope this helps as you migrate to Nuxt 3. ❤️

**Reply by Austio** [reply]
Thank you Daniel
